src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[\u2713]" : "[\u2718]"); //return [tick] or [X] symbols
```

Comment:
Have you considered using constant variable to store the symbols? It will help improve readability (knowing what it is at a glance), and allow easy editing of values if needed